tools/snowflake/spcs_dashboard/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG BASE_IMAGE=python:3.11.9-slim-bullseye
+FROM $BASE_IMAGE
+
+COPY ./ /trulens_dashboard/
+
+WORKDIR /trulens_dashboard
+
+RUN pip install -r requirements.txt
+RUN pip install trulens_connectors_snowflake-1.0.1-py3-none-any.whl
+RUN pip install trulens_dashboard-1.0.1-py3-none-any.whl
```

Comment:
This is temporary, once the next version of trulens is released, then this can go in requirments.

Reply:
Check with corey whether requirements.txt can be done using poetry instead.